tajo-plan/src/main/java/org/apache/tajo/plan/verifier/PreLogicalPlanVerifier.java

```diff
@@ -354,4 +354,14 @@ public Expr visitInsert(Context context, Stack<Expr> stack, Insert expr) throws
 
     return expr;
   }
+
+  // TODO: This should be removed at TAJO-1891
+  @Override
+  public Expr visitAlterTable(Context context, Stack<Expr> stack, AlterTable expr) throws TajoException {
+    if (expr.getAlterTableOpType() == AlterTableOpType.ADD_PARTITION) {
+      context.state.addVerification(new UnsupportedException("ADD PARTITION"));
```

Comment:
It should be UnimplementedException. UnsupportedException means there is no plan to support this feature.